tools/dev/import-dev-keys.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-# Requires bash version 3 or so for regular expression pattern match
-
-COMMITTER_KEYS_URL='https://raw.githubusercontent.com/wiki/rapid7/metasploit-framework/Committer-Keys.md'
+COMMITTER_KEYS_URL='https://raw.githubusercontent.com/rapid7/metasploit-framework/master/docs/metasploit-framework.wiki/Committer-Keys.md'
```

Comment:
I thought this has now been moved over to https://docs.metasploit.com/docs/development/maintainers/committer-keys.html? We should probably deduplicate this data.

Reply:
@gwillcox-r7 `https://raw.githubusercontent.com/rapid7/metasploit-framework/master/docs/metasploit-framework.wiki/Committer-Keys.md` is used to generate `https://docs.metasploit.com/docs/development/maintainers/committer-keys.html`

Reply:
Woops mistake on my part, marking resolved.

Reply:
What Alan said. :)

Reply:
(And the official docs mention this broken script.)

> Altering this table’s layout will almost certainly break [import-dev-keys.sh](https://github.com/rapid7/metasploit-framework/blob/master/tools/dev/import-dev-keys.sh), so please don’t.